opennebula/shared_schemas.go

```diff
@@ -353,6 +353,15 @@ func generateVMTemplate(d *schema.ResourceData, tpl *vm.Template) {
 		tpl.AddOS(vmk.Boot, osconfig["boot"].(string))
 	}
 
+	//Generate CPU Model definition
+	cpumodel, ok := d.Get("cpumodel").([]interface{})
```

Comment:
`cpumodel` does not exist for VM temapltes while this functon is also called in that case.

that is why everything linke to this feature must be wrote in "sharedSchema .go" directly

Reply:
Added in vm templates